### PR TITLE
Include csv.icon field in testpmd-lb-operator and cnf-app-mac-operator

### PR DESCRIPTION
**Pull request by ramperher**
Found this issue in DeployableByOLM and ValidateOperatorBundle preflight tests on cnf-app-mac-operator and testpmd-lb-operator:

```
# cnf-app-mac-operator
err="the bundle cannot be deployed because deployment validation has failed: [{Name:cnf-app-mac-operator.v0.2.16-202405211327.4010727 
Errors:[Error: Value : (cnf-app-mac-operator.v0.2.16-202405211327.4010727) csv.Spec.Icon elements should contain both data and mediatype]

# testpmd-lb-operator
err="the bundle cannot be deployed because deployment validation has failed: [{Name:testpmd-lb-operator.v0.2.14-202405211327.4010727
Errors:[Error: Value : (testpmd-lb-operator.v0.2.14-202405211327.4010727) csv.Spec.Icon elements should contain both data and mediatype]
```

This is working in the other two operators.

The icon refers to an image that identifies the operator, that can be found in the [operator metadata](https://github.com/operator-framework/operator-lifecycle-manager/blob/4197455/Documentation/design/building-your-csv.md#operator-metadata). We provide it in the base CSV provided in the manifests for testpmd-operator and trex-operator, but not in cnf-app-mac-operator and testpmd-lb-operator.

Including a new icon image, which is smaller than the current one, in all operators.

This should fix the errors to make DeployableByOLM and ValidateOperatorBundle preflight tests pass on these operators.

- [x] The two target preflight tests are now passing, e.g. [testpmd-operator](https://www.distributed-ci.io/jobs/1d7794e5-be4a-4931-a5c8-49bb1599513f/tests/d058a5a1-02c9-4830-9fdb-61d261e6d72a). This is ready for review.

### CODE REVIEW

**Comment by ramperher** [comment]
Testing right now, CSV has now the icon field:

```
$ oc get csv -n example-cnf cnf-app-mac-operator.v0.2.16-pr84.73987836 -o json
...
        "description": "Operator to monitor namespace for PODs with VF to get MAC address of VF",
        "displayName": "CNF App MAC Fetch Operator",
        "icon": [
            {
                "base64data": "iVBORw0KGgoAAA...
...
```

Same for testpmd-lb-operator:

```
$ oc get csv -n example-cnf testpmd-lb-operator.v0.2.14-pr84.73987836 -o json
...
       "description": "LoadBalancer for FastData path traffic",
        "displayName": "LoadBalancer",
        "icon": [
            {
                "base64data": "iVBORw0KGgoAAAA...
...
```

Let's wait for preflight's execution, but now the result should be different.

**Comment by ramperher** [comment]
> from change #84:
> 
> * FAILURE https://www.distributed-ci.io/jobs/4919b5d3-0c87-4572-a1c7-3f1aad70ff0e/jobStates

The failure is not related to the change, it is failing because verify-tests was expecting cnf-certification-test JUnit file but I didn't launch certsuite, that's why it failed here.

In fact, the two target tests for this change are now passing in the target operators:

- testpmd-lb-operator - https://www.distributed-ci.io/jobs/4919b5d3-0c87-4572-a1c7-3f1aad70ff0e/tests/f74572d3-4442-4cc4-932a-4f66e3d5f55c
- cnf-app-mac-operator - https://www.distributed-ci.io/jobs/4919b5d3-0c87-4572-a1c7-3f1aad70ff0e/tests/2c9aaba3-1420-4c53-ba00-3bf873574477

This change is ready for review

File: cnf-app-mac-operator/config/manifests/bases/cnf-app-mac-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ spec:
...
...ogucYFIDXwJC92qvIuR19S8sPduBMrKc/1t3+v6QA5SD5Qs5WNXcUnSSUj26KhgCQyIkcJRyLZ5MSBAQOjaQjVh6a10pN427bhpP9FqpGlSzStFZLATDKS0luF9+xoEBKiHBnAQeM6WQICD0o/YqwotdbZ52nIWbRnMTvi9XVp5vmub6UKsGxynB8HK5aR77ylY5CF0oP+PAQJUBvhcmMpj1wKqDp7MQEkZdZRIQelC+ZGpvq+k9xJSbxjw34fmJtx3dKe1Eu2VjltHKN81N01woVZKxXXTnbWFVrzQFnF6p5I69AvpGWXFw1NefNi200L401gApIPSn9irCpS43TztJvikubUcvTCwo5M/6UgkGo2gnOq1cJSkX3TEEhfz+X8vBZ2rnEabgUAV0TDeD86r1YOe6bVoZAN4ZY4AUEHpSbhxUEU6Qj9FCUBhz6W6xYnBj6dWJKBfdxaAwtPL2fnnfF8rnAEZsoTV26AMbk6la96EMcl4e2yCngNCv2m8izqeUOt087TRKULhaWo+GeON4lP1Sxn1+ihWDk8yDQmk9emkAIzH3SuXn3BjniwDHWxjYmO/vM6SgcHMhGLgGtezQIOfgKwoCQo+GUkXoevO00yqpfH7j+Jny5TakL+P98p4/U24mrxmtOVmuqJSRmBwOX6lopYgH5WYgb1i3M+XKD3BwfbpWKgqvVDyQ9aAMtOXBqauCQfdKUKjxOnacB+VeJQ/QHVzbUtM0kVLKafjVY/7BjWiaJtX0ftpSVgt6u/K3+VrNLRMppctlBYH85/mlv7BZ+YsgfwHftopNO0pgnZ/z3R7P+fxc3lh3o7qUUrP05AaM5fvUtWrzUkr5d+JSRW/pThm13biUUq4qXK3g2rTWd1ct53aMv1Nlv6nF69jW0l/qx4MyGT2f6/yzcfe4gcsPAgIMVfml2y3l3vznxR4/yn75BZv/ot3WbtK9cs53Fs75hRYuxg/mX5bz8+lcAutauDbNr09dXpvuLHx3uQ4NQAmTO+X61dZ1bG5+b/Kw/JkDwP1Vq0cCAqNTfvEuLPzyzXeKXmdkZN4S9vDQL5wbyMqUC/L8XOdzf1Kr3PyL8v5xoycAbSodBPPvqXPle2pu54hR5XvlmjN3vzwObvx8f41LqZgv/lw8qTp2d+FnpNX7EgEBAAB4JCL+fwRBcOn9x9KJAAAAAElFTkSuQmCC
```
Comment (tonyskapunk) [inline cnf-app-mac-operator/config/manifests/bases/cnf-app-mac-operator.clusterserviceversion.yaml:22]: Please consider downloading a smaller icon from https://www.redhat.com/en/about/brand/standards/icons/standard-icons (Download icons (Red Hat credentials required)) There you will find a small favico that is only 4KB

Comment (ramperher) [reply cnf-app-mac-operator/config/manifests/bases/cnf-app-mac-operator.clusterserviceversion.yaml:22]: Ack, I'll update all images then, thanks

**Comment by ramperher** [comment]
Testing. New image looks good, e.g. from cnf-app-mac-operator's CSV:

```
$ oc get csv -n example-cnf cnf-app-mac-operator.v0.2.17-pr84.5525412b -o json
...
        "icon": [
            {
                "base64data": "iVBORw0KGgoAAAANSUhEUgAAACAAAAAgEAYAAAAj6qa3AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAGYktHRAAAAAAAAPlDu38AAAAJcEhZcwAABCoAAAQqAKd4QIUAAAAHdElNRQfoAwcSCQrCAVRvAAAAAW9yTlQBz6J3mgAAB+RJREFUaN7tmWtwVdUVx38nyc0lgVBySQiFJKQYUARBUB4mYsWEUkNBq0IAtUAptji2lY6WUobO6DCttjOSlKnWNhEKpoylUEkJQ2spbQ3ESmhUwiNgoAiBGMy5eZKE5Ob0w3+fmDD9kHtDnM6U/WXd/Vh7rfVfa6+99rlwo/1/NytURv9BUWcEAEMYDEAdV82CTwwNAODx3Q5Ae2/3t5sBCDcaBmgxE82GRhu5syTXdzQ0O8JCBcAZC8BABhjDLYDEB4kAmH9W1HPYSGj3z+glsF9wBRjD2wAi7qAT4MtN2i8xl0gj9z2jR4gtaAD8Qt5iEADNxAPE3CqDX0rXqgUHqAZwhjHcMJYZfuu/y/V7zfilHgDAeQAnQf3s/QJ84+2SF7PVREazfdzoFWQLmsE/2+j3SzMQD/BUlUJ9aa188Ug2HQCxW3Ukxv5FHtuRQTiAM83ZaPhbjSJreqj1vkJ9wb/wApy6jSYA+w31dx6W5ltfkdxN48xRw5cQnD1BR4BTZH4kAcTPVWg+niy6aYJoU4sAyE2U58aPphjAWUGDMfFpQ3/QU5OwJgBnmc74+Bjtl+uX55t/pnWbPFr92Fo8APE55igG3XoNgH+5+XGlO506SAaGvyUP792gkFw8SQA4ZxSqOXcxDYCjkpiSRz3A6jfl2dVvyKMpeZ3hAJRpv5yZZp8EeXixjyiAvZbWh8/V+NTPuRHgf6SfACDtGgDCACZukIcq36QWoGGuAMn8jRQqLNK8P1njw18Xc/4eAZNVqiORdUwhn79ZfMNzJcefIP7C01qX+br4GlZrXeUR9SfOc48Sd/YTAM5S8yPC0ABAfKkMq6nRBRe5VgrHvSOFT78rD4Lo9Nlan1goA5c1aJ9lrRof+Q/tP/0W16MK8dN/ksfjKjUeuVaTNXGSG19gkjLOY/0EAB8ZGukiAtC+UoZGZunMBvKkaMcyKR49s4vfAqh7XBI9HypZzoiUwTPqzObvKWnWLekJXHSmOu0LxB/4neR7X5X8qylunWCN6icAOlquGQgAnH1ZiozKlic6/6zJioWid5dTada3ApSsVoSUXBLfr/4tA/OOiL/kLCcBSl7VkXI1vPuMOqfSqQLoNAE/6mnNn80SEBB2uZ8ACHeTixGk6+hwrjqJi+TJ1LcVIdu/K4MySkgBSJui5BVoUCR4h5hbY7+AzK8Wf9Sj3AMQ+KOyetqDAipjnyJj+3piAcYskuYjT0v+4Zcw90LnB/0EwNCKawDwAJQnq3MiSkfgiTIpXvycdt5tCZDsq6KTLgqA0aagLdgiuq1c4zcnyfMTdxg+48/dxwVAcYuAfSJTehxv1dk/ts49mrH3BgdA8IWQBHmcGgDa5cl0Ex/5X5ThP+q0kgAK73Q+BojZo5J2ZYwkTqqzPg+wpM05ARBRL0N2ZWj+7Z0mMh62RgA0ZjiXAB6oleHPvSb69UoBc6iQRunlG2306mWL6O1Ct5kKrt0NOSWfg7+X4TkxOtNZx5wygN2Xdd9fiZOHMieIb/O3nQsAgVMmqRmVi85qfoFfof7zXU4FQOfHMnj2DlNyv8YegEPjrUeNXnZwhve5ua81u9H0WwHCqu0qgCFHbZPF7SaAWZtsP8C7ttYnLuziq+5OUw/YtQAfvCx6z012S/f9B//W7wEIe9E2pvpFwgmxhfwc7gJCCkaYe7vDLXW7VIoE2HhInaR9XABYOJBhAJ1ruh49quxuUwT84biOVsX3rZEAz/7EqTfr3MiTvAhfpJEbYgv5Oew2X5QUsH7a3WDXoIQlMmSWKZDS0mT4mhdkeHSKrscBPoX2UxPEN7lY9L4K5zxA3P6uN4S5d/pq+HUDwG2mAgvvquC8ADML1Gm5ovFn98mD8+cpQvJbVRDlrRUgS3Jl0vpR0qz9IwGaXulGlPMNI+c6tesGAEMACEjhsOky6IEfSsKBKo0X5MjAX39P419tlCnZJ0S3bdDjaMsIzf/dRND8k0ZKjaEB/lea/1ui9kVDLwCkbrc/ASiLsesBMkptG+A7EUqG5ePsGoB1y5UM15XYdQAn47T+mTuUPO/3iq90iuZTsE3Wsxu4Li3oa/Da5uiZG971CWsgQOYZ9Qc8I/rkNnk4+RZiANbEKRKKNjt6Zm+2XgF4f5fGn68XX9UcSRmcr5xw70qNb1ll3ibuUQg5IkL/Juhm74eNAkMBvLuk6LyVmhy7V6q175Rhi8fp3ir6W5cC7geWAMCeh9RZZGr9tirtN6ZeR2HedB0lTyrJfTO8zwD4dX9bXaVHA8CEg1J0/GR5an203vkrvilJH+5AtXqE5TNAntc+5pNYmGLy1CpVisu3yMfPn9N+k1eo8Lr5Vjf/27X0qYV8BCx9FXYcNz1FA3xlmhQtzdSb4EWPAAlkijKQdwBojm0yfDHaxz9F1LhkkOMHaPyFAPzxVC2eek773D9St0y5e+269YzDZ9Vs893fvgwQe5eSU3GtktPX5tj13eeDB9pWZES4HlYyXFWt/v4j2n/Q0r4mw9CvQTf0rwIkme8xjR3yyF8fck22ngRCKViGGb5O0+8EeGuOfByZrtOf4u1r9R9yKWzXdN/BW6AjMPSf+iJz8QVjcnRsXd8UtM+ZHw5A2JcE8BhHr8vzpsK8cp/vps8aAIW21+zQpg8eYMWbBcrPVmzbp+qHJEdH4dO/yNyLz/1CpQiJ8qX2GL3RbrRetv8ABTa0S/ux8acAAAAASUVORK5CYII=",
                "mediatype": "image/png"
            }
        ],
...
```

Running preflight tests.

**Comment by ramperher** [comment]
Preflig

> from change #84:
> 
> * SUCCESS https://www.distributed-ci.io/jobs/1d7794e5-be4a-4931-a5c8-49bb1599513f/jobStates

The two target preflight tests are now passing, e.g. [testpmd-operator](https://www.distributed-ci.io/jobs/1d7794e5-be4a-4931-a5c8-49bb1599513f/tests/d058a5a1-02c9-4830-9fdb-61d261e6d72a). This is ready for review.

**Comment by ramperher** [comment]
In the last commit, I just updated the operator version and rebase from main, no new tests are needed.

**Comment by ramperher** [comment]
Same, my last commit is not a code change, just removing a deprecated task in the dockerfile, not required to launch a new job

**Review by manurodriguez — APPROVED** [review]
Thanks, LGTM

**Comment by tonyskapunk** [comment]
mini-icon :+1:

**Pull request merged**